src/commands/vote-mockup.ts

```diff
@@ -44,30 +60,32 @@ export default class VoteMockupCommand extends SlashCommand<Client> {
 
     await ctx.fetch();
 
-    const ballot = new Map<string, string>();
-
     ctx.registerComponent('vote', async (selectCtx) => {
-      if (game.players.findIndex((p) => p.id === selectCtx.user.id) === -1) {
-        selectCtx.send('You are not in this game.', { ephemeral: true });
+      console.log(`Received vote with ${start + duration - Date.now()}ms remaining`);
+
+      if (game.players.findIndex((p) => p.id === selectCtx.user.id) === -1)
+        return selectCtx.send('You are not in this game.', { ephemeral: true });
+
+      if (start + duration < Date.now()) {
```

Comment:
```suggestion
      if (start + duration < selectCtx.invokedAt) {
```
[ComponentContext#invokedAt: `number`](https://slash-create.js.org/#/docs/main/latest/class/ComponentContext?scrollTo=invokedAt)